### PR TITLE
Feature/305 Add monitoring layers to all Community tabs

### PR DESCRIPTION
**Pull request by maxdiebold-erg**
## Related Issues:
* [HMW-305](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=864&view=detail&selectedIssue=HMW-305)

## Main Changes:
* Added the _Current Water Conditions_ and _Past Water Conditions_ layers to the layer list on all tabs on the Community page.

## Steps To Test:
1. Confirm that the layers appear in the layer on all tabs
2. The layers should not initially be visible, except on the Monitoring sub-tab

### CODE REVIEW

**Review by cschwinderg — CHANGES_REQUESTED** [review]
Looks good!

I noticed one thing. When you expand the `Watershed Health Scores` on the Protect panel, the monitoring location layers should be hidden. We already do this with the `allWaterbodiesLayer`.

File: app/client/src/components/pages/Community.Tabs.Protect.js
Comment (cschwinderg) [inline app/client/src/components/pages/Community.Tabs.Protect.js:299]: The `useCurrentValue` parameter can be removed, as it looks like it isn't being used.

**Review by cschwinderg — CHANGES_REQUESTED** [review]
Thank you for fixing that. 

I found one more small item, listed below.

File: app/client/src/components/pages/Community.Tabs.Protect.js
Comment (maxdiebold-erg) [reply app/client/src/components/pages/Community.Tabs.Protect.js:299]: Hmm that was originally there, and it's used on line 374, so I'll test if it's needed. But the `newValues` doesn't need to be an array, I can switch that back.

Comment (cschwinderg) [reply app/client/src/components/pages/Community.Tabs.Protect.js:299]: Oops, I missed that one. Sorry about the confusion.

Don't worry about testing it, everything is good as is. I'll approve and merge once this other build completes.

**Review by cschwinderg — APPROVED** [review]

**Comment by maxdiebold-erg** [comment]
@cschwinderg Shoot I didn't get back to this before it was merged, I actually need to either revert those `newValues` fields or change them all to that, but line 1207 still uses `key` and `newValue`. Sorry, I'd been halfway through trying that method and didn't realize they were still there on the last commit. I can open a new PR with that.

**Comment by cschwinderg** [comment]
@maxdiebold-erg No worries. Just open up a new PR.

**Pull request merged**
